From 33e855255d4e2986412d1b8aa47a904e1a01fe88 Mon Sep 17 00:00:00 2001
From: Michael Pauzner <mpauzner@habana.ai>
Date: Tue, 02 Jan 2024 11:23:08 +0200
Subject: [PATCH] [FSW-1764] common: include updated

Header file hl_fw_if.h has split in two files.
Thus, the include is updated.

Change-Id: I46146d8b1218ea073d190c8ef2563400d9a89c99
Signed-off-by: Michael Pauzner <mpauzner@habana.ai>
---

diff --git a/common/boot_desc.h b/common/boot_desc.h
index 0d0ccc6..b66de88 100644
--- a/common/boot_desc.h
+++ b/common/boot_desc.h
@@ -7,7 +7,7 @@
 #ifndef _BOOT_DESC_H_
 #define _BOOT_DESC_H_
 
-#include <hl_fw_if.h>
+#include <hl_fw_upd_if.h>
 #include <hl_boot_desc_rom.h>
 #include <sha512.h>
 
